WooCommerce/WooCommerceTests/ViewRelated/HubMenu/HubMenuViewModelTests.swift

```diff
@@ -29,6 +29,17 @@ final class HubMenuViewModelTests: XCTestCase {
         XCTAssertFalse(viewModel.menuElements.contains(.inbox))
     }
 
+    func test_menuElements_do_not_include_payments_when_feature_flag_is_off() {
```

Comment:
Should we go the other way around with this test, so we do not need to remove it once this feature goes live? Something like `test_menuElements_include_payments_when_feature_flag_is_on` instead?

Reply:
Actually I think both cases are important, when the feature flag is on and off. I added the case you mentioned in 1320c6331219eb23329d465e60ab228fb142abe2